doc/modules/reference.rst

```diff
@@ -336,7 +336,7 @@ the :ref:`user guide <user_guide>` for more information and usage examples.
    plot_stat_map
    plot_glass_brain
    plot_connectome
-   plot_connectome_strength
```

Comment:
Both functions should be in the reference API while they're still in the code base !